### PR TITLE
fix: Skip generating cdn/deploy with ./build.sh <action> --debug

### PR DESCRIPTION
**Pull request by Meng-Heng**
Follows https://github.com/keymanapp/keyman.com/pull/471 and implements onto keymanweb.

This PR is ready for review.

### CODE REVIEW

File: resources/init-container.sh [hunk 1]
```diff
@@ -1,7 +1,12 @@
 #!/usr/bin/env bash
 
-echo "---- Generating CDN ---"
-rm -rf cdn/deploy
-cd cdn
-php -d include_path=/var/www/html/_includes:. cdnrefresh.php
-cd ..
+if [[ ! $1 =~ "debug"]]; then
```
Comment (darcywong00) [inline resources/init-container.sh:3]: I think the bash needs whitespace before the closing bracket
```suggestion
if [[ ! $1 =~ "debug" ]]; then
```

**Comment by Meng-Heng** [comment]
Thanks, @darcywong00!

File: resources/init-container.sh [hunk 2]
```diff
@@ -1,7 +1,12 @@
...
+if [[ ! $1 =~ "debug" ]]; then
+    echo "---- Generating CDN ---"
+    rm -rf cdn/deploy
+    cd cdn
+    php -d include_path=/var/www/html/_includes:. cdnrefresh.php
+    cd ..
+else 
+    echo "Skip Generating CDN and clean CDN cache"
+    rm -rf cdn/deploy
+fi
```
Comment (Meng-Heng) [inline resources/init-container.sh:12]:
```suggestion
    cd ../cdn
    php -d include_path=/var/www/html/_includes:. cdnrefresh.php
    cd ..
else 
    echo "Skip Generating CDN and clean CDN cache"
    rm -rf ../cdn/deploy
fi
```

**Review by darcywong00 — APPROVED** [review]
lgtm

**Pull request merged**